internal/venice-common/src/main/java/com/linkedin/venice/views/VeniceView.java

```diff
@@ -1,22 +1,22 @@
 package com.linkedin.venice.views;
 
+import com.linkedin.venice.exceptions.VeniceException;
 import com.linkedin.venice.meta.Store;
 import com.linkedin.venice.utils.VeniceProperties;
 import java.util.Collections;
 import java.util.Map;
-import org.apache.avro.Schema;
-import org.apache.kafka.clients.consumer.ConsumerRecord;
+import java.util.Properties;
 
 
 public abstract class VeniceView {
-  protected final VeniceProperties props;
+  protected final Properties props;
   protected final Store store;
+  protected final Map<String, String> viewParameters;
 
-  public VeniceView(VeniceProperties props, Store store) {
+  public VeniceView(Properties props, Store store, Map<String, String> viewParameters) {
```

Comment:
Why we change `VeniceProperties` to `Properties`? This line `new Class<?>[] { VeniceProperties.class, Store.class, Map.class } ` in `ViewUtils` might not work.

Reply:
I swapped it out because the VeniceWriterFactory needed it as Properties and it struck me that that could be a more robust/broader interface for future implementations.